src/components/addList.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import './list.css'
+import Popup from 'reactjs-popup';
+import PropTypes from 'prop-types'
+
+function AddItem(props){
```

Comment:
You should use object destructuring, while showing props for component.

Something like this...
```js
function AddItem({item : { firstName, lastName }, itemsList }) {

....
}

```

Reply:
if I destructure like this, how to change the below statement
 props.addtoList([...props.itemsList])